### ISSUE TITLE
Is Laravel 9 support planned?

### ISSUE THREAD

**Issue by axcherednikov** [php-mqtt/laravel-client#21]

**Comment by Namoshek** [comment]
Added support for Laravel 9 in [`v1.0.2`](https://github.com/php-mqtt/laravel-client/releases/tag/v1.0.2).

**Comment by axcherednikov** [comment]
I never managed to install it. Pure Laravel 9

![Снимок экрана 2022-02-15 в 22 10 56](https://user-images.githubusercontent.com/52137648/154132139-d2d034af-d2c5-4013-bcc6-65015c488afe.png)

**Comment by Namoshek** [comment]
Mh, I remember it being not straight forward to add `psr/log:^3.0` to `php-mqtt/client` because it's a breaking change in the `Logger`. Let me see what I can do...

**Comment by axcherednikov** [comment]
Thank you!

**Comment by Namoshek** [comment]
Realizing that the `Logger` has been marked as `internal` by me, but also seeing that `@return void` is documented, I think it is safe to add the `void` return type to the `Logger` methods, which makes this a safe and non-breaking change in `php-mqtt/client`:

```php
/**
 * Wrapper for another logger. Drops logged messages if no logger is available.
 *
 * @internal This class is not part of the public API of the library and used internally only.
 * @package PhpMqtt\Client
 */
class Logger implements LoggerInterface
{
    /**
     * System is unusable.
     *
     * @param string $message
     * @param array  $context
     * @return void
     */
    public function emergency($message, array $context = []): void   // <-- the ': void' is new
    {
        $this->log(LogLevel::EMERGENCY, $message, $context);
    }
}
```

**Comment by Namoshek** [comment]
I released [`php-mqtt/client:1.1.3`](https://github.com/php-mqtt/client/releases/tag/v1.1.3), should be available as composer package shortly.
